test/integration/serviceaccount/legacy_service_account_token_clean_up_test.go

```diff
@@ -181,14 +181,13 @@ func TestLegacyServiceAccountTokenCleanUp(t *testing.T) {
 				} else {
 					t.Logf("The secret %s has been labeled with %s", test.secretName, lastUsed)
 				}
-			}
-
-			_, err = c.CoreV1().Secrets(myns).Get(context.TODO(), test.secretName, metav1.GetOptions{})
-			if err != nil {
-				if apierrors.IsNotFound(err) {
-					t.Fatalf("The secret %s should not be cleaned up, err: %v", test.secretName, err)
-				} else {
-					t.Fatalf("Failed to get secret %s, err: %v", test.secretName, err)
+				_, err = c.CoreV1().Secrets(myns).Get(context.TODO(), test.secretName, metav1.GetOptions{})
```

Comment:
Moving this `Get` into this `if` statement has no value - the exact same `.Get` is performed on [line 174](https://github.com/kubernetes/kubernetes/blob/06cdf35b5da878489618820798cf1ac2b5f6931f/test/integration/serviceaccount/legacy_service_account_token_clean_up_test.go#LL174C23-L174C105).